src/plugins/condition/components/Condition.vue

```diff
@@ -182,11 +182,6 @@ export default {
             outputOptions: ['false', 'true', 'string']
         };
     },
-    computed: {
-        initCap: function (string) {
-            return string.charAt(0).toUpperCase() + string.slice(1)
-        }
```

Comment:
You do not pass arguments to computed properties. initCap should be a method. We also make extensive use of underscore in our codebase. I don't really know it that well but I know it has this functionality built-in as _.capitalize(string). This is really nit-picky but if you are concerned with capitalization and you want to keep the method you might consider string.slice(1).toLowerCase().

Reply:
Deep helped clarify that for me yesterday and I made it a method in a later commit on a different branch.